api/index.ts

```
@@ -0,0 +1,5 @@
+import { handle } from "@hono/node-server/vercel";
+// eslint-disable-next-line ts/ban-ts-comment
+// @ts-expect-error
+import app from "../dist/src/app.js";
```

ESLint

[error] Do not import modules in `dist` folder, got ../dist/src/app.js

(antfu/no-import-dist)